### PR TITLE
Added isaac@stackhpc.com as new user

### PR DESCRIPTION
**Pull request by Wasaac**

### CODE REVIEW

File: etc/kayobe/users.yml [hunk 1]
```diff
@@ -20,6 +21,15 @@ users_default:
   # Authorise all users' SSH keys for the kolla user. This allows them to
   # operate kolla.
   - "{{ lookup('file', kayobe_config_path ~ '/ssh-keys/id_rsa_bharat.pub') }}"
+  - "{{ lookup('file', kayobe_config_path ~ '/ssh-keys/id_rsa_isaac.pub') }}"
+
+- username: isaac
```
Comment (oneswig) [inline etc/kayobe/users.yml:26]: Why the Isaac user here?

**Review by oneswig — DISMISSED** [review]
It doesn't appear there is a precedent for defining the `isaac` user here, do we have a reason for doing so?

File: etc/kayobe/users.yml (hunk 1 above)
Comment (Wasaac) [reply etc/kayobe/users.yml:26]: Copying the convention for setting up users as per https://github.com/RSE-Cambridge/cumulus-kayobe-config/blob/rocky/etc/kayobe/users.yml

Presumably other users have been added manually here.

**Review by dougszumski — APPROVED** [review]
Looks good to me. I can apply this now.

**Pull request merged**